var/exp_repo/experiments/amg2023/experiment.py

```diff
@@ -0,0 +1,175 @@
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+
+
+class Amg2023(Experiment):
+    variant(
+        "programming_model",
+        default="openmp",
+        values=("openmp", "cuda", "rocm"),
+        description="on-node parallelism model",
+    )
+
+    variant(
+        "workload",
+        default="problem1",
+        description="problem1 or problem2",
+    )
+
+    variant(
+        "experiment",
+        default="example",
+        values=("strong", "weak", "example"),
+        description="type of experiment",
+    )
+
+    #TODO: Support list of 3-tuples
+    #variant(
+    #    "p",
+    #    description="value of p",
+    #)
+
+    #TODO: Support list of 3-tuples
+    #variant(
+    #    "n",
+    #    description="value of n",
+    #)
+
+    def compute_include_section(self):
+        return ["./configs/software.yaml",
+                "./configs/variables.yaml",
+                "./configs/modifier.yaml",]
```

Comment:
@pearce8 @scheibelp @becker33 `include ./configs` does not work for modifier.py

Reply:
> @pearce8 @scheibelp @becker33 `include ./configs` does not work for modifier.py

@rfhaque Is this still an issue?